### DISCUSSION TITLE
Why this repository was changed  from github/hub?

### DISCUSSION THREAD

**Discussion by yutkat** [github/hub#3326] [Q&A] [answered]
This repository should have been github/hub not too long ago. Why was it moved back under the private repository?

**Comment by mislav** [comment, marked as answer]
I was the only person ever working on the hub project, even though it was "owned" by the `github` org on GitHub. I've recently left GitHub as a place of employment, and have requested that they transfer the hub project to my account so I can continue maintaining it.
